src/key-logging.ts

```diff
@@ -21,7 +21,7 @@ export const setupKeyLoggingInspector = async (
   page: Page,
   eventDataHandler: (event: BlacklightEvent) => void,
 ) => {
-  page.on("request", (request: Request) => {
```

Comment:
Why are we removing the type here?

Reply:
Request type is no longer exported by Puppeteer.

Reply:
might be better to replace it with this https://pptr.dev/#?product=Puppeteer&version=v14.1.0&show=api-class-httprequest

Reply:
Good idea, done!